### PR TITLE
Funding statement

### PR DESCRIPTION
**Pull request by agitter**
I added a funding statement, sorting grants alphabetically and associating author initials with their respective awards.

### CODE REVIEW

File: author_contact_approve.md [hunk 1]
```diff
@@ -1,28 +1,28 @@
 | GitHub Handle | Full name | ORCID  | Affiliation(s)  | Email | Approve | Grants to Cite |
 |---------------|-----------|--------|-----------------|-------|---------|----------------|
 | cgreene | Casey S. Greene | 0000-0001-8713-9213 | Department of Systems Pharmacology and Translational Therapeutics. Perelman School of Medicine. University of Pennsylvania. | csgreene@upenn.edu | Yes | Gordon and Betty Moore Foundation GBMF 4552 |
```
Comment (cgreene) [inline author_contact_approve.md]: y

File: author_contact_approve.md [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
-| j3xugit | Jinbo Xu | 0000-0001-7111-4839 | Toyota Technological Institute at Chicago | jinboxu@gmail.com | Yes | NIH R01GM089753,  NSF/BIO-1564955 |
+| j3xugit | Jinbo Xu | 0000-0001-7111-4839 | Toyota Technological Institute at Chicago | jinboxu@gmail.com | Yes | NIH R01GM089753, NSF 1564955 |
```
Comment (cgreene) [inline author_contact_approve.md]: y

File: author_contact_approve.md [hunk 3]
```diff
@@ -1,28 +1,28 @@
...
 | gailrosen | Gail L. Rosen | 0000-0003-1763-5750 | Ecological and Evolutionary Signal-processing and Informatics Laboratory. Department of Electrical and Computer Engineering. Drexel University | gailr@ece.drexel.edu | Yes | NSF 1245632 |
```
Comment (cgreene) [inline author_contact_approve.md]: y

File: author_contact_approve.md [hunk 4]
```diff
@@ -1,28 +1,28 @@
...
-| davharris | David J. Harris | 0000-0003-3332-9307 | Wildlife Ecology and Conservation | dave@harris-research.me | Yes  | Gordon and Betty Moore Foundation GBMF 4563 |
-| bdo311 | Brian T. Do | 0000-0003-4992-2623 | Harvard Medical School | brian_do@hms.harvard.edu | Yes | NIGMS T32GM007753  |
+| davharris | David J. Harris | 0000-0003-3332-9307 | Wildlife Ecology and Conservation | dave@harris-research.me | Yes | Gordon and Betty Moore Foundation GBMF 4563 |
```
Comment (cgreene) [inline author_contact_approve.md]: y

File: author_contact_approve.md [hunk 5]
```diff
@@ -1,28 +1,28 @@
...
+| bdo311 | Brian T. Do | 0000-0003-4992-2623 | Harvard Medical School | brian_do@hms.harvard.edu | Yes | NIH T32GM007753 |
```
Comment (cgreene) [inline author_contact_approve.md]: y

File: author_contact_approve.md [hunk 6]
```diff
@@ -1,28 +1,28 @@
...
 | gwaygenomics | Gregory P. Way | 0000-0002-0503-9348 | Department of Systems Pharmacology and Translational Therapeutics. Perelman School of Medicine. University of Pennsylvania. | gregway@upenn.edu | Yes | |
 | alxndrkalinin | Alexandr A. Kalinin | 0000-0003-4563-3226 | Department of Computational Medicine and Bioinformatics. University of Michigan Medical School. | akalinin@umich.edu | Yes | |
-| agitter | Anthony Gitter | 0000-0002-5324-9833 | Department of Biostatistics and Medical Informatics, University of Wisconsin-Madison. Morgridge Institute for Research | gitter@biostat.wisc.edu | Yes | NIH/NIAID U54AI117924 |
-| brettbj | Brett K. Beaulieu-Jones | 0000-0002-6700-1468 | Genomics and Computational Biology Graduate Group, Perelman School of Medicine. University of Pennsylvania. | brettbe@upenn.edu | Yes | NIH AI116794 |
+| agitter | Anthony Gitter | 0000-0002-5324-9833 | Department of Biostatistics and Medical Informatics, University of Wisconsin-Madison. Morgridge Institute for Research | gitter@biostat.wisc.edu | Yes | NIH U54AI117924 |
```
Comment (cgreene) [inline author_contact_approve.md]: y

File: author_contact_approve.md [hunk 7]
```diff
@@ -1,28 +1,28 @@
...
+| brettbj | Brett K. Beaulieu-Jones | 0000-0002-6700-1468 | Genomics and Computational Biology Graduate Group, Perelman School of Medicine. University of Pennsylvania. | brettbe@upenn.edu | Yes | NIH R01AI116794 |
```
Comment (cgreene) [inline author_contact_approve.md]: y

File: author_contact_approve.md [hunk 8]
```diff
@@ -1,28 +1,28 @@
...
 | blengerich | Benjamin J. Lengerich | 0000-0001-8690-9554 | Computational Biology Department, School of Computer Science, Carnegie Mellon University | blengeri@cs.cmu.edu | Yes | |
 | agapow | Paul-Michael Agapow | 0000-0003-1126-1479 | Data Science Institute, Imperial College London | p.agapow@imperial.ac.uk | Yes | |
 | enricoferrero | Enrico Ferrero | 0000-0002-8362-100X | Computational Biology and Stats, Target Sciences, GSK | enrico.x.ferrero@gsk.com | Yes | |
 | annecarpenter | Anne E. Carpenter | 0000-0003-1555-8261 | Imaging Platform, Broad Institute of Harvard and MIT | anne@broadinstitute.org | Yes | NIH R01GM089652 |
```
Comment (cgreene) [inline author_contact_approve.md]: y

File: author_contact_approve.md [hunk 9]
```diff
@@ -1,28 +1,28 @@
...
-| evancofer | Evan M. Cofer | 0000-0003-3877-0433 | Department of Computer Science. Trinity University. | ecofer@princeton.edu | Yes | NSF MRI-ACI-1531594 |
+| evancofer | Evan M. Cofer | 0000-0003-3877-0433 | Department of Computer Science. Trinity University. | ecofer@princeton.edu | Yes | NSF 1531594 |
```
Comment (cgreene) [inline author_contact_approve.md]: y

File: author_contact_approve.md [hunk 10]
```diff
@@ -1,28 +1,28 @@
...
 | DaveDeCaprio | Dave DeCaprio | 0000-0001-8931-9461 | ClosedLoop.ai | daved@alum.mit.edu | Yes | |
 | akundaje | Anshul Kundaje | 0000-0003-3084-2287 | Dept. of Genetics, Dept. of Computer Science, Stanford University | akundaje@stanford.edu | Yes | NIH DP2GM123485 |
```
Comment (cgreene) [inline author_contact_approve.md]: y

File: author_contact_approve.md [hunk 11]
```diff
@@ -1,28 +1,28 @@
...
 | sw1 | Stephen Woloszynek | 0000-0003-0568-298X | Department of Electrical and Computer Engineering, Drexel University | sw424@drexel.edu | Yes | |
 | yfpeng | Yifan Peng | 0000-0001-9309-8331 | National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health | yifan.peng@nih.gov | Yes | National Institutes of Health Intramural Research Program and National Library of Medicine |
 | dhimmel | Daniel S. Himmelstein | 0000-0002-3012-7446 | Department of Systems Pharmacology and Translational Therapeutics. Perelman School of Medicine. University of Pennsylvania. | daniel.himmelstein@gmail.com | Yes | Gordon and Betty Moore Foundation GBMF 4552 |
```
Comment (cgreene) [inline author_contact_approve.md]: y

**Comment by cgreene** [comment]
Should be! Anyone else who gets added will be responsible for modifying the funding statement accordingly.

**Comment by agitter** [comment]
Actually, I'd rather monitor it myself.  I'll take responsibility for keeping it up to date.

**Pull request merged**